jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/NonexistentServerUrlDockerCredentialHelperException.java

```diff
@@ -16,11 +16,13 @@
 
 package com.google.cloud.tools.jib.registry.credentials;
 
+import java.nio.file.Path;
+
 /** Thrown because the credential helper does not have credentials for the specified server URL. */
 public class NonexistentServerUrlDockerCredentialHelperException extends Exception {
 
   NonexistentServerUrlDockerCredentialHelperException(
```

Comment:
This name is also a bit hard to parse.  `UnknownServerUrlException`?  Maybe the docker stuff should be in a separate package?

Reply:
ditto #832